cypress/e2e/tests/migration/applicationinventory/applications/crud.test.ts

```diff
@@ -49,4 +64,22 @@ describe(["@tier1"], "Application crud operations", () => {
         // Assert that newly created application is deleted
         notExists(application.name);
     });
+
+    it("Update application profile details", function () {
```

Comment:
This test will be very similar to https://github.com/konveyor/tackle-ui-tests/blob/main/cypress/e2e/tests/migration/applicationinventory/applications/interlinked.test.ts#L67 .

TC 1 is "Create application and validate info on application profile page"
TC2 is " Edit application from application profile page"